api/storage_service.cc

```diff
@@ -464,13 +464,14 @@ void set_sstables_loader(http_context& ctx, routes& r, sharded<sstables_loader>&
         boost::algorithm::to_lower(primary_replica);
         bool load_and_stream = stream == "true" || stream == "1";
         bool primary_replica_only = primary_replica == "true" || primary_replica == "1";
+        auto scope = parse_stream_scope(req->get_query_param("scope"));
```

Comment:
Somewhere (here or in the sstables_loader::load_new_sstables) there must be a check that load-and-stream is given too. API is called not by nodetool only, so scylla need to sanitize its arguments as well